### ISSUE TITLE
RemoveStore, RemoveIndex, and RemoveIndexGroup commands can remove multiple entities

### ISSUE THREAD

**Issue by emacthecav** [locationtech/geowave#1192]
Multiple similarly named entities can all be removed by one of the removal commands. In `AbstractRemoveCommand.java` the patterns are dealt with via the following

```		
for (final String key : existingProps.stringPropertyNames()) {
			if (key.startsWith(pattern)) {
				keysToRemove.add(key);
			}
		} 
```

The fact that it only checks the start of the entity names means that stores/indexes that start with the same letters can be removed by one call (i.e `index1` and `index2` would both be removed by calling `geowave config rmindex index`)

**Comment by rfecher** [comment]
Covered by PR #1190

**Comment by rfecher** [comment]
merged with PR #1190
